rspace/src/main/scala/coop/rchain/rspace/LMDBStore.scala

```diff
@@ -2,55 +2,69 @@ package coop.rchain.rspace
 
 import java.nio.ByteBuffer
 import java.nio.file.Path
-import java.security.MessageDigest
-import scala.collection.immutable.Seq
 
-import coop.rchain.rspace.internal._
-import coop.rchain.rspace.internal.scodecs._
 import coop.rchain.rspace.util._
 import org.lmdbjava.DbiFlags.MDB_CREATE
 import org.lmdbjava._
-import scodec.Codec
-import scodec.bits._
+import scodec.bits.BitVector
 
-import scala.collection.JavaConverters._
+import scala.collection.AbstractIterator
 
 /**
   * The main store class.
   *
   * To create an instance, use [[LMDBStore.create]].
   */
-class LMDBStore[C, P, A, K] private (env: Env[ByteBuffer],
-                                     _dbKeys: Dbi[ByteBuffer],
-                                     _dbWaitingContinuations: Dbi[ByteBuffer],
-                                     _dbData: Dbi[ByteBuffer],
-                                     _dbJoins: Dbi[ByteBuffer])(implicit
-                                                                sc: Serialize[C],
-                                                                sp: Serialize[P],
-                                                                sa: Serialize[A],
-                                                                sk: Serialize[K])
-    extends IStore[C, P, A, K]
-    with ITestableStore[C, P] {
+case class LMDBStore[C, P, A, K](env: Env[ByteBuffer])(implicit
```

Comment:
Personal taste: I wouldn't make it a `case class`.

Reply:
Agreed - this shouldn't be a case class.